internal/keycloakb/configdbauthtransaction.go

```diff
@@ -0,0 +1,83 @@
+package keycloakb
```

Comment:
Separation of duty between this file and configdbmodule is unclear to me. I would suggest to move all the authorizations queries is a dedicated file whcih would also contains the mechanism for transactions

Reply:
ConfigDBModule is a module for queries without transactions... Requests are executed directly on a DB connection instance.
When creating a DB transaction, you want to execute requests on the transaction, not directly on the db connection.
Alternatives could be renaming configdbauthtransaction to configdbtransaction (it could execute other requests in a transaction, not only authorizations)... or have an instance of ConfigDBModule that wraps a first one (like context.Context), the wrapped one would use the transaction, the non-wrapped one would call directly the DB without transaction

Reply:
It seems to me there is some Transaction management in ConfigDBModule. Maybe we can see it more as batch queries ?